bin/setup_dev

```diff
@@ -62,8 +62,8 @@ $AG install \
     ca-certificates sudo net-tools tcpdump build-essential \
     isc-dhcp-client network-manager netcat gnupg2 strace \
     python3 python3-pkg-resources python3-setuptools python3-dev \
-    python3-pip python emacs-nox python3-venv libffi-dev \
-    python3-cairocffi jq pandoc
+    python3-pip openjdk-8-jdk python emacs-nox python3-venv \
```

Comment:
this should not be returning

Reply:
Do you mean the openjdk-8-jdk package?
I had removed it and then it created a conflict that I resolved by re-adding.
So confusing! I will remove it again then!

Reply:
Done